var/spack/repos/builtin/packages/jemalloc/package.py

```diff
@@ -24,22 +24,24 @@ class Jemalloc(Package):
 
     variant('stats', default=False, description='Enable heap statistics')
     variant('prof', default=False, description='Enable heap profiling')
-    variant('je', default=False, description='Prepend the public API functions with "je_"')
+    variant(
+        'jemalloc_prefix', default='none',
+        description='prefix',
```

Comment:
Could we make this description a little more descriptive? Something like:
```suggestion
        description='prefix to prepend to all public APIs',
```

Reply:
Whoops. That was a placeholder string that should have been changed before I pushed.

Fixed with 79a917f9f61717c27728dd04baff00ea5d1286e7